mixer/adapter/kubernetesenv/config/config.proto

```diff
@@ -69,15 +69,15 @@ message Params {
     // Default: 5 minutes
     google.protobuf.Duration cache_refresh_duration = 2 [(gogoproto.nullable)=false,(gogoproto.stdduration) = true];
 
-    // Namespace for secret created for multi-cluster support.
+    // Namespace of the secret created for multi-cluster support.
     //
-	// Details on multi-cluster and the kubernetes secret required to
-	// access the remote cluster's credentials can be found here:
-	// https://istio.io/docs/setup/kubernetes/multicluster-install
-	//
-    // NOTE: If cluster_registries_namespace is not set then the environmental
-    // variable POD_NAMESPACE will be checked/used. If POD_NAMESPACE is not
-    // set then cluster_registries_namespace will default to "istio-system".
+    // Details on multi-cluster and the kubernetes secret required to
+    // access the remote cluster's credentials can be found here:
+    // <https://istio.io/docs/setup/kubernetes/multicluster-install>
```

Comment:
Please make this a markdown link `[Multicluster Install](....)`

Reply:
Done